static/openapi.yaml

```diff
@@ -1455,10 +1461,10 @@ components:
               type: string
               example: https://cdn.modrinth.com/data/AABBCCDD/b46513nd83hb4792a9a0e1fn28fgi6090c1842639.png
               description: The URL of the project's icon
-              nullable: true
```

Comment:
This is nullable

Reply:
Really? Does not seem like that from the source code, if there is no icon the returned string seems to be empty.

Reply:
Yup: https://api.modrinth.com/v2/project/dungeon_quest

Reply:
Interesting, its just an empty string in the search results: <https://api.modrinth.com/v2/search?query=Dungeon%20Quest>